src/helpers/helpers.test.ts

```diff
@@ -0,0 +1,39 @@
+import { isArgFormatValid } from './helpers';
```

Comment:
Ooooooooooooooh, my previous comment must have confused you. I was actually looking at ***`app`*** `/src/helpers/helpers.test.ts` 😄. Alright, well, your code is even better (as more future-proof) with a `describe`.